Cargo.toml

```diff
@@ -86,8 +86,9 @@ structopt = "0.3.15"
 chrono = "0.4.11"
 
 # Substrate
```

Comment:
Except for the `serde` bit, I'm not exactly sure why there are changes to `Cargo.toml`. Can you elaborate on them?
Otherwise everything looks good.

Reply:
Yea you are correct, I had copied the structure of `#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]` and added some unnecessary libraries for this use case i'll make a push that would remove them.